new-lamassu-admin/src/pages/Customers/components/IdDataCard.js

```diff
@@ -14,6 +14,21 @@ import { getName } from '../helper'
 
 import Field from './Field'
 
+const formatGender = field => {
+  // In US cards, gender comes as a number. In BC and ZA cards, gender comes as a letter
+  const genderMap = {
+    1: 'M',
+    2: 'F'
+  }
```

Comment:
I would rather have this in the machine code alongside the specific parser for US.